### ISSUE TITLE
Method misused.

### ISSUE THREAD

**Issue by etherge** [sensu-plugins/sensu-plugins-pagerduty#21]
In bin/handler-pagerduty.rb, a method called _resolve_ is called to resolve incident.

```
pagerduty.get_incident([incident_key_prefix, incident_key].compact.join('')).resolve(
            description: [description_prefix, event_summary].compact.join(' '), details: @event)
```

The _resolve_ method is defined in lib pagerduty like this:

```
def resolve(description = nil, details = nil)
    modify_incident("resolve", description, details)
 end
```

Notice that _resolve_ method is **not** defined with **Ruby keyword parameter**, but it is invoked with **keyword parameter**.  Actually, the _description_ and _details_ are not treated as **two** parameters but treated as **one** Hash type parameter.

This issue will cause Pagerduty incidents would not be resolved when the length of description is greater than 1024. And Pagerduty does not return an error when description is passed as Hash type( will be convert to Json object when sending to Pagerduty as payload).

**Comment by heffergm** [comment]
Any chance of getting a release cut with this fix?

**Comment by eheydrick** [comment]
Released in 1.0.0!
